### ISSUE TITLE
why  len(aug_tensor) = 6?

### ISSUE THREAD

**Issue by xiaocheng2333** [Luchixiang/PCRL#3]
`def aug(self, img):
        img = img.numpy()
        aug_tensor = [0 for _ in range(6)]
        if random.random() < 0.5:
            img = np.flip(img, axis=1)
            aug_tensor[0] = 1
        if random.random() < 0.5:
            img = np.flip(img, axis=2)
            aug_tensor[1] = 1
        times = int(random.random() // 0.25)
        img = np.rot90(img, k=times, axes=(1, 2))
        aug_tensor[times + 2] = 1
        return torch.tensor(img.copy(), dtype=torch.float), torch.tensor(aug_tensor)
`

**Comment by xiaocheng2333** [comment]
In the paper, len(vec(T )) = 7.

**Comment by Luchixiang** [comment]
For 3D image len(vec(T)) = 7 while for 2D images len(vec(T)) = 6
